studio_flow/production.json

```diff
@@ -0,0 +1,1918 @@
+{
+  "description": "Production - Mutual Aid Call Center",
+  "states": [
+    {
+      "name": "Trigger",
+      "type": "trigger",
+      "transitions": [
+        {
+          "event": "incomingMessage"
+        },
+        {
+          "next": "say_hello",
+          "event": "incomingCall"
+        },
+        {
+          "event": "incomingRequest"
+        }
+      ],
+      "properties": {
+        "offset": {
+          "x": 490,
+          "y": -1340
+        }
+      }
+    },
+    {
+      "name": "phone_tree_1",
+      "type": "gather-input-on-call",
+      "transitions": [
+        {
+          "next": "split_key_press",
+          "event": "keypress"
+        },
+        {
+          "event": "speech"
+        },
+        {
+          "next": "phone_tree_2",
+          "event": "timeout"
+        }
+      ],
+      "properties": {
+        "play": "https://firebrick-sloth-5004.twil.io/assets/phone-tree-one.mp3",
```

Comment:
What is this firebrick-sloth-5004? Is this the magical place that files get placed at when you drag mp3's into Studio?

Reply:
not 1,000% if it is dedicated to us, but essentially this is our URL of `public` twilio assets
but the part I'm uncertain on is if https://firebrick-sloth-5004.twil.io is dedicated to us or is shared